impl-base/src/test/java/org/jboss/shrinkwrap/impl/base/ShrinkWrapTestCase.java

```diff
@@ -365,32 +359,51 @@ public void shouldBeAbleToimportZipFileViaArchiveFactory() throws Exception {
      *
      * @throws Exception
      */
-    @Test(expected = IllegalArgumentException.class)
+    @Test
     public void importFromNonZipFileThrowsException() throws Exception {
-        final File nonZipFile = new File(TestSecurityActions.getThreadContextClassLoader()
-            .getResource(NAME_FILE_NON_ZIP).toURI());
-        ShrinkWrap.createFromZipFile(JavaArchive.class, nonZipFile);
+        InputStream inputStream = TestSecurityActions.getThreadContextClassLoader()
```

Comment:
It would be much easier to move the file to this module's resources

Reply:
That was a quick temporary solution which I missed to repair. Sorry about that.